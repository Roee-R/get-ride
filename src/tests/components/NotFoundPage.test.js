import React from 'react';

import expenses from '../fixtures/expenses';
import {shallow} from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

describe('Not found page test', ()=>{
    test('should render not found page', ()=>{
        const wraper = shallow(<NotFoundPage />)
        expect(wraper).toMatchSnapshot();
    })
})

