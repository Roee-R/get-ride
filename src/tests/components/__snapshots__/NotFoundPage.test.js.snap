// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page test should render not found page 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
